Code/Query/Query.h

```diff
@@ -140,16 +141,19 @@ class Query {
   };
 
  protected:
+  MatchFuncArgType d_val = 0;
```

Comment:
I'm going to have to remember that this form of initialization of data members is legal in C++11.